nservicebus/sagas/index.md

```diff
@@ -188,11 +188,11 @@ The recommended approach is for the saga to publish events containing the requir
 
 ## Saga state read/write behavior
 
-Saga state is read just before `Handle` is invoked and written immediately after `Handle` is completed. In other words, the sequence is read, invoke, and write per `Handle`.
+Saga state is read immediately before a message processing method is invoked, and written immediately after the method returns. The sequence of read, invoke, and write occurs once per message processing method.
```

Comment:
I changed `Handle` to "message processing method" because it could be a `Timeout` method.